File: C++/Two_Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> result;

        for (vector<int>::iterator p = begin(nums); p != end(nums)-1; p++)	{
            vector<int>::iterator j = find(p+1, end(nums), (target - *p));

            // Found
            if (j != end(nums)) {
                result = { p-begin(nums), j-begin(nums) };
                break;
            }
        }
        return result;
    }
};

File: Python/Two_Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for index in range(len(nums)-1):
            search = target-nums[index]
            search_list = nums[index+1:len(nums)]
            if search in search_list:
               index2 = search_list.index(search)+index+1
               break
        return [index, index2]